api.py

```diff
@@ -0,0 +1,116 @@
+import os
+import threading
+import enum
+from time import sleep
+from subprocess import Popen, DEVNULL
```

Comment:
just import `subprocess`, and then use the stuff we want like `subprocess.Popen` and `subprocess.DEVNULL`.

this helps people unfamiliar with our code know what were using is from the `subprocess`module